AISKU/package.json

```diff
@@ -52,7 +52,10 @@
         "@microsoft/applicationinsights-common": "2.4.0-beta",
         "@microsoft/applicationinsights-core-js": "2.4.0-beta",
         "@microsoft/applicationinsights-dependencies-js": "2.4.0-beta",
-        "@microsoft/applicationinsights-properties-js": "2.4.0-beta",
+        "@microsoft/applicationinsights-properties-js": "2.4.0-beta"
+    },
+    "peerDependencies": {
+        "@microsoft/applicationinsights-rollup-es3" : "1.0.0",
```

Comment:
Does this need to be a peer dependency? Would it not work as a dev dependency?

Reply:
Moved as suggested